Formula/msgpack-cpp.rb

```diff
@@ -0,0 +1,54 @@
+class MsgpackCpp < Formula
+  desc "MessagePack implementation for C++ / msgpack.org[C++]"
+  homepage "https://msgpack.org/"
+  url "https://github.com/msgpack/msgpack-c/releases/download/cpp-4.0.1/msgpack-cxx-4.0.1.tar.gz"
+  sha256 "acba76ebc3ff9af2fd2649f319b25cd2387c66f75ae74a7a652fe3386e45d498"
+  license "BSL-1.0"
+  head "https://github.com/msgpack/msgpack-c.git", branch: "cpp_master"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+
+  conflicts_with "msgpack", because: "both install `msgpack` binaries"
```

Comment:
I think the conflict comes from both installing `include/msgpack`. Perhaps we can rename the one here to `include/msgpack-cxx`?
```ruby
include.install include/"msgpack" => "msgpack-cxx"
```
That we we won't need the conflict.

Reply:
ok, let's do that.

Reply:
The out-of-the-box example does not work with this change.